extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -28,16 +28,18 @@ export function registerTableDesignerCommands(appContext: AppContext) {
 			}
 			const tableIcon = context.nodeInfo!.nodeSubType as azdata.designers.TableIcon;
 			const telemetryInfo = await getTelemetryInfo(context, tableIcon);
+			let nonDefaultOptions = await azdata.connection.getNonDefaultOptions(context.connectionProfile);
 			await azdata.designers.openTableDesigner(sqlProviderName, {
 				title: NewTableText,
-				tooltip: `${context.connectionProfile!.serverName} - ${context.connectionProfile!.databaseName} - ${NewTableText}`,
+				tooltip: context.connectionProfile!.connectionName ? `${context.connectionProfile!.connectionName} - ${NewTableText}` : `${context.connectionProfile!.serverName} - ${context.connectionProfile!.databaseName} - ${NewTableText}`,
 				server: context.connectionProfile!.serverName,
 				database: context.connectionProfile!.databaseName,
 				isNewTable: true,
 				id: generateUuid(),
 				connectionString: connectionString,
 				accessToken: context.connectionProfile!.options.azureAccountToken as string,
-				tableIcon: tableIcon
+				tableIcon: tableIcon,
+				additionalInfo: `${context.connectionProfile!.serverName + ' - ' + context.connectionProfile!.databaseName}${nonDefaultOptions}`
```

Comment:
Does this show up pretty clearly or should there be some separation between the database name and non default options?

Reply:
nonDefaultOptions already has the space added in front.